resources/assets/js/app.js

```diff
@@ -13810,8 +14064,72 @@ var APPModalMTDetails = function (_APPModal) {
 
 				// title
 				$panel.append('<div class="panel-heading">' + menu.label + '</div>');
-                var $plot = $(modelHandler._img).appendTo($panel).wrapAll('<div class="panel-plot"></div>');
-				//var $plot = $('<figure class="figure"><img src="' + modelHandler._img + '" /></figure>').appendTo($panel).wrap('<div class="panel-plot"></div>');
+
+				var $plot = $(modelHandler._img).appendTo($panel).wrapAll('<div class="panel-plot"></div>');
+			}
+
+			return $panel;
+		}
+		/**
+   * CREATE PLOT PANEL
+   * create plot tab-pane for specific menu
+   * @param {array} menu
+   * @param {object} modelHandler: object of class Model
+   */
+
+	}, {
+		key: '_createModelScriptPanel',
+		value: function _createModelScriptPanel(menu, modelHandler) {
```

Comment:
_createModelScriptPanel and _createVisualizationScriptPanel are duplicated. The only difference is the script used in the function. They could be replaced with a parameterised _createScriptPanel. This can also help us if we ever include the readme.

Reply:
we keep it in mind